nextroute/check/schema.go

```
@@ -0,0 +1,202 @@
+package check
+
+import "strings"
+
+// Verbosity is the verbosity of the check.
+type Verbosity int
+
+const (
+	// Off does not run the check.
+	Off Verbosity = iota
+	// Low checks if there is at least one move per plan unit.
+	Low
+	// Medium checks the number of moves per plan unit and the
+	// number of vehicles that have moves. It also reports the number of
+	// constraints that are violated for each plan unit if it does not fit
+	// on any vehicle.
+	Medium
+	// High is identical to medium.
+	High
+	// VeryHigh reports for each plan unit which vehicle have moves.
+	VeryHigh
+)
+
+// ToVerbosity converts a string to a verbosity. The string can be
+// anything that starts with "o", "l", "m", "h" or "v" case-insensitive.
+// If the string does not start with one of these characters the
+// verbosity is off.
+func ToVerbosity(s string) Verbosity {
```

Aren't we being to flexible here? Wouldn't it be better to now swallow user errors (typos) and instead make sure the string is actually correct?